extensions/jdbc/jdbc-postgresql/runtime/src/main/java/io/quarkus/jdbc/postgresql/runtime/graal/SQLXLMFeature.java

```diff
@@ -0,0 +1,56 @@
+package io.quarkus.jdbc.postgresql.runtime.graal;
+
+import java.lang.reflect.Method;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+import org.graalvm.nativeimage.hosted.Feature;
+import org.graalvm.nativeimage.hosted.RuntimeReflection;
+
+import com.oracle.svm.core.annotate.AutomaticFeature;
+
+@AutomaticFeature
+public final class SQLXLMFeature implements Feature {
+
+    private final AtomicBoolean triggered = new AtomicBoolean(false);
+
+    /**
+     * To set this, add `-J-Dio.quarkus.jdbc.postgresql.graalvm.diagnostics=true` to the native-image parameters
+     */
+    private static final boolean log = Boolean.getBoolean("io.quarkus.jdbc.postgresql.graalvm.diagnostics");
```

Comment:
Any reason you're not using... you know, actual loggers?

Reply:
I've been wondering if I should, but consider this is only ever possibly printed during the native-image output - during which we don't expose logging configuration nor options to redirect output to anything else than stdout.

I guess I'd keep it simple for now - and this is mimicking a precedent from `CacheConstructorsAutofeature` in the Caffeine module.

Most likely our next step is to have some standard template / helper to unify such approaches, at that point we can consider using a logger - but maybe just KISS?